.github/workflows/python-client.yml

```diff
@@ -33,24 +33,26 @@ jobs:
     strategy:
       matrix:
         go: [ '1.18' ]
+        python: [ '3.8', '3.9', '3.10' ]
     steps:
       - uses: actions/checkout@v2
       - uses: ./.github/workflows/go-setup
       - uses: ./.github/workflows/python-tests
         with:
-          python-version: '3.8'
+          python-version: ${{ matrix.python }}
           path: 'client/python'
   python-client-integration-tests:
     runs-on: ubuntu-22.04
     strategy:
       matrix:
         go: [ '1.18' ]
+        python: [ '3.8', '3.9', '3.10' ]
```

Comment:
Thanks for taking the initiative on adding these to integration tests. I actually think I’d prefer to just leave this with just 3.8 for now.  

Your code is correct but the container that runs these integration tests is actually 3.8.  Let’s hold off on running integration tests for other versions for now.

Reply:
Yes I thought so. Will try to revert it before I leave